docs/disk-encryption-set.md

```diff
@@ -32,7 +32,7 @@ Follow [tutorial-create-cluster](https://docs.microsoft.com/en-us/azure/openshif
 ```bash
 export KEYVAULT_NAME=$USER-enckv
 export KEYVAULT_KEY_NAME=$USER-key
-export DISK_ENCRYPTION_SET_NAME=$USER-des
+export DISK_ENCRYPTION_SET_NAME=$RESOURCEGROUP-disk-encryption-set
```

Comment:
Does this doc assume `$RESOURCEGROUP` is already set?

Reply:
Yes, we assume a reader will check ARO tutorial where this is mentioned (among all the other assumptions):

> Follow [tutorial-create-cluster](https://docs.microsoft.com/en-us/azure/openshift/tutorial-create-cluster) but don't run the az aro create command, instead proceed as follows ...